src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/VisualState_Tests.cs

```diff
@@ -119,7 +119,7 @@ public void TestListViewReleasedOut()
 
 		[Test]
 		[AutoRetry]
-		[Ignore("Test is flaky on WASM")] // WASM-only test, but very flaky #9080
+		[ActivePlatforms(Platform.Browser)] // WASM-only test, but very flaky #9080
```

Comment:
If this helps, then it should be ok to remove the comment here too

Reply:
Does not look look like it was, let's review later.